fontbe/src/glyphs.rs

```diff
@@ -110,46 +116,119 @@ fn create_composite(
     Ok(composite.unwrap())
 }
 
+/// * <https://github.com/fonttools/fonttools/blob/3b9a73ff8379ab49d3ce35aaaaf04b3a7d9d1655/Lib/fontTools/ttLib/tables/_g_l_y_f.py#L335-L367>
+/// * <https://docs.microsoft.com/en-us/typography/opentype/spec/tt_instructing_glyphs#phantoms>
+fn add_phantom_points(advance: u16, points: &mut Vec<Point>) {
+    // FontTools says
+    //      leftSideX = glyph.xMin - leftSideBearing
+    //      rightSideX = leftSideX + horizontalAdvanceWidth
+    // We currently always set lsb to xMin so leftSideX = 0, rightSideX = advance.
+    points.push(Point::new(0.0, 0.0)); // leftSideX, 0
+    points.push(Point::new(advance as f64, 0.0)); // rightSideX, 0
+
+    // TODO: vertical phantom points
+    points.push(Point::new(0.0, 0.0));
+    points.push(Point::new(0.0, 0.0));
+}
+
+fn point_seqs_for_simple_glyph(
```

Comment:
Would it make sense for these to be methods on `ir::Glyph`? I guess we don't own that type, here?

Reply:
Interesting question. I think I would argue IR shouldn't directly implement translation to BE constructs so no.